openstudiocore/src/energyplus/ForwardTranslator.cpp

```diff
@@ -355,6 +355,14 @@ Workspace ForwardTranslator::translateModelPrivate( model::Model & model, bool f
     globalGeometryRules.setString(openstudio::GlobalGeometryRulesFields::RectangularSurfaceCoordinateSystem, "Relative");
     m_idfObjects.push_back(globalGeometryRules);
 
+    {
+      auto zoneAirContaminantBalances = model.getConcreteModelObjects<ZoneAirContaminantBalance>();
```

Comment:
Thats a good question. Yes I tested that it now works and you only get one instance in the idf file. The question is was it already working before i did this and if it wasn't then why not.

I generally don't like that catch all, but this seems like a good application of it.